app/instance-initializers/bugsnag.js

```diff
@@ -22,6 +23,12 @@ export default {
       Ember.onerror = function(error) {
         Bugsnag.context = getContext(router);
         const metaData = getMetaData(error, owner);
+        const user = getUser(owner);
+
+        if (user && typeof user === 'object') {
```

Comment:
What is the scenario where `user` would not exist or where it wouldn't be an object? `app/utils/bugsnag-get-user.js` always returns and object, correct?